update coc evolution (#7)
yyjtech/code-of-conduct: README.md

# YYJ Tech Code of Conduct

YYJ Tech strives to be as equitible and inclusive as possible. This code of conduct sets the expectations and guidelines for all members on how to conduct themselves to create a community that is as welcoming and inclusive as possible, as well as the consequences for unacceptable behavior.

However, we are all humans, and we know that there are times that we will not get things right. 

> Help us out by sharing things you are concerned about here. You can find more information about how to do that in the [Reporting Guidelines](https://github.com/yyjtech/code-of-conduct/blob/master/reporting_guidelines.md).

Reviewer: I read "things you are concerned about here" as if it were a link (ie like "click [here](https://some.place.where.i.go.to.give.feedback.com) to share things you are concerned about".  Maybe this could be worded as:

> Help us out by sharing things you are concerned about on YYJ Tech.   You can find more information about how to do that in the [Reporting Guidelines](https://github.com/yyjtech/code-of-conduct/blob/master/reporting_guidelines.md)

## 1\. Purpose

A primary goal of YYJ Tech is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).

We invite all those who participate in the YYJ Tech community to help us create safe and positive experiences for everyone.

## 2\. Open Tech Citizenship

A supplemental goal of this Code of Conduct is to increase open tech citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their impacts on our community.

Communities mirror the societies in which they exist and positive action is essential to counteract the many forms of inequality and abuses of power that exist in society.

> If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to celebrate and support that.

Reviewer: Oooh, nice, I *really* like this change to the wording.

Reviewer 2: Plus 1

> Behave as if you were at a professional event

Although we are on the internet - and perhaps even more importantly _because_ we are on the internet where tone is extremely challenging to read - choosing our words carefully and conducting yourself as a professional is important. Not sure if what you're about to say is professional? Try imagining someone in your family or friend group who's got your back, but holds you to high standards - whether that be your grandma, your cousin, our best friend or your mom - would they be cheering on what you're about to say, or would they be disappointed in your level of professionalism? 

> Use kindness to build community

This doesn't mean we don't hold each other to high standards for behaviour, but it does mean we call each other in (or out, if necessary) on things we are uncomfortable with in a way that is grounded in empathy and respect - even if we don't feel like we ourselves were respected in that moment. 

> > Be curious to hear about the other person’s perspective

Reviewer: 💯

Remember not to assume your experience is the default - make space for others to have different life experiences than you. One litmus test for how you are framing something is going to read as having implicit bias is to check against these three questions: 

> * *First: is this true* is what you're about to say something that is a fact, or a perspective that you hold where you can acknolwedge that others may hold different ones? Is there a way to explicitly acknolwedge that you're open to other perpsectives in how you're responding; will this read as a "yes and", or a "no because"? 

Reviewer: Minor revision:

```suggestion
* *First: is this true?*  Is what you're about to say something that is a fact, or a perspective that you hold where you can acknowledge that others may hold different ones? Is there a way to explicitly acknowledge that you're open to other perspectives in how you're responding; will this read as a "yes and", or a "no because"? 
```

Author: thank you, my fingers hate that word lol

> * *Second: Is this missing something* are you encouraging all possibilities, or just the ones that come most readily to you and your life experience? What questions might you ask the other folks in the conversation to understand their experiences or motivations better? 

Reviewer: ```suggestion
* *Second: Is this missing something?*  Are you encouraging all possibilities, or just the ones that come most readily to you and your life experience? What questions might you ask the other folks in the conversation to understand their experiences or motivations better? 
```

> * *Third: is this just* is this supporting the inclusion and equality of minority members of our community, or reinforcing barriers for those groups? This can be quite nuanced, but being explicit about who and how we want to advocate for each other, and why it matters to us, is always a great reminder. 

Reviewer: ```suggestion
* *Third: is this just?*  Is this supporting the inclusion and equality of minority members of our community, or reinforcing barriers for those groups? This can be quite nuanced, but being explicit about who and how we want to advocate for each other, and why it matters to us, is always a great reminder. 
```

TBH I'm struggling with the word `just` in that, but I can't think of a better word. 🤷  Maybe `fair`, but that doesn't have as much weight.

Reviewer 2: Yeah, I had to read it a few times, because I was reading just as **only**, as opposed to **equitable**. 

I wonder if this could be improved by replacing `this` with `your framing`. 

It does make each of the questions in the litmus more wordy, but I feel like it may add clarity.

Author: This part was a combination of [this article](https://dailystoic.com/dont-make-assumptions/) and one of the suggested responses to the prompts during the interviews that was similar in nature (the is it fair/is it kind/is it true). 

I agree the word "just" is weird, but I also appreciate the nod to justice/equality. I like your suggestion jamie, and I'm pushing up an overall modification to this paragraph. Please let me know if it reads better!

> Take the time to understand the motivations behind what the other person has share, rather than drawing your own conclusions. When in doubt: repeat back what you think you're hearing, and ask why the topic matters to them. 

Reviewer: typo:

```suggestion
Take the time to understand the motivations behind what the other person has shared, rather than drawing your own conclusions. When in doubt: repeat back what you think you're hearing, and ask why the topic matters to them. 
```

> We all have to help to foster safer and braver spaces

> Different opinions are welcome in YYJ Tech, but context matters greatly. Please do your best to read the room and not derail the purpose(s) of the channels you are in, and respect the boundaries of others if they ask something of you. 

Reviewer 2: Do we want to change `channels` to `channels and threads`? I feel like I've seen places where they were on topic, but morphed a thread in a way that probably didn't make it the most inviting place for the original poster...

Author: changing to "discussions" so it's more inclusive

Figuring out how to include someone who does not have the same experience as you is not easy, and if we succeed in being a welcoming place that will mean we are interacting with many more people with many diverse experiences. Don't be afraid to try, but be open to feedback and don't take offense if someone asks you to do something differently to include them. Work together to find a way to make it possible. 

**The following behaviors are expected and requested of all community members:**

* Participate in a respecful and active way that supports these community values above. In doing so, you contribute to the health and longevity of this community.

> * Exercise self-reflection and empathy in your speech and actions. Tone on the internet is often misunderstood, and people may interpret your words differently than you might intend. So err on the side of caution, and include the why behind your thoughts as often as possible.

Reviewer 2: by `the why behind your thoughts` are we talking `the reasoning behind your thoughts` or something else?

* Attempt collaboration before conflict.

> * Refrain from demeaning, discriminatory, or harassing behaviors and speech.

Reviewer 2: For consistency, should we use `behaviours`? I think you have behaviours above.

Author: good catch

* Be mindful of your surroundings and of your fellow participants. [Alert community organizers](https://goo.gl/forms/94uxzsNnwDLIotQ23) if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
* Remember that community event venues may be shared with members of the public; please be respectful to all patrons of these locations.

### Examples 

| Situation |  Not acceptable | Why this is not ideal | Better options |
|-----------|-----------------|-----------------|-----------------|

> | **You see someone you want to connect with, but you don't already know or have a good relationship with** | You send them a message asking for something from them, and keep messaging them until they respond| Think of a DM like invading someone's personal space - ask politely to be invited in, like you would coming into a home or how you might introduce yourself at an in-person networking event before starting to talk.| - Reply to the thread where you initially ran across them, and ask if they would like to connect via DM or LinkedIn <br> - Ask someone you mutually know to introduct you <br> - Request to add them on LinkedIn <br> - Start your message with "Hello, I hope it's ok to reach out via DM..."|

Reviewer: ```suggestion
| **You see someone you want to connect with, but you don't already know or have a good relationship with** | You send them a message asking for something from them, and keep messaging them until they respond| Think of a DM like invading someone's personal space - ask politely to be invited in, like you would coming into a home or how you might introduce yourself at an in-person networking event before starting to talk.| - Reply to the thread where you initially ran across them, and ask if they would like to connect via DM or LinkedIn <br> - Ask someone you mutually know to introduce you <br> - Request to add them on LinkedIn <br> - Start your message with "Hello, I hope it's ok to reach out via DM..."|
```

| **You see a post that you're not sure is on topic for the group, and you're extra suspicious because this person doesn't seem to be on a BC timezone** | Ask "is this Victoria related?" or "Are you from Victoria"? | Many people use this Slack group as a way of seeking employment to come to Victoria or Canada, not only because they already live here. Reacting this way makes people feel unwelcomed in the community. Imagine coming to a public event and being asked "do you belong here?" - that wouldn't feel great. | - Welcome! What are you hoping to get out of this group?" Perhaps you can direct them to the right resources, or clear up confusion about the purpose of the community if there is any <br> - Type `!welcome` to trigger a welcome message that can help guide to find the part of the community they are looking for <br> - Hello! It looks like you aren't currently located in Victoria. Are you planning to move here? You should check out https://www.movingtovictoria.com/ - it's a great resource curated by one of our community members. |

> | **You see a community member or organization behaving in a way that you think could be better** | "Well that's stupid! It should be this way instead..." | This kind of response mimics how we often respond online to things super disconnected from us (e.g. politics, media) - but in this case, these are real members of our community who are sharing, and we may work with them in future, collaborate on community projects, or connect at events. | - If you believe what that community member was doing violated community standards, please flag for moderators via the [reporting form](https://goo.gl/forms/94uxzsNnwDLIotQ23) <br> - Open with language that sets up that you remember there is a human receiving your feedback, like "Hi _poster_..." or "Hey really appreciate this post, but I was wondering..." <br> - If you've got something potentially hard to hear, start with "Hey there, wondering if you're open to some feedback on this post?" This prepars the other person for feedback, and lets them choose the format and timing of the feedback so they can be in the appropriate headspace to receive it. |

Reviewer 2: ```suggestion
| **You see a community member or organization behaving in a way that you think could be better** | "Well that's stupid! It should be this way instead..." | This kind of response mimics how we often respond online to things super disconnected from us (e.g. politics, media) - but in this case, these are real members of our community who are sharing, and we may work with them in future, collaborate on community projects, or connect at events. | - If you believe what that community member was doing violated community standards, please flag for moderators via the [reporting form](https://goo.gl/forms/94uxzsNnwDLIotQ23) <br> - Open with language that sets up that you remember there is a human receiving your feedback, like "Hi _poster_..." or "Hey really appreciate this post, but I was wondering..." <br> - If you've got something potentially hard to hear, start with "Hey there, wondering if you're open to some feedback on this post?" This prepares the other person for feedback, and lets them choose the format and timing of the feedback so they can be in the appropriate headspace to receive it. |
```

Some topics also necessitate more professionalism, kindness, and sensitivity to minority experiences than others. For example, channels like #job-postings or #events that are more focused on experiences related to professional networking have a stronger need to adhere to and uphold these norms than more casual social channels like #watercooler, #politics or #random. 

Community members are strongly encouraged to help by modelling and pointing to these community ideals for others when they see tensions arise, or flag for moderators if they do not feel comfortable themselves. Moderators will actively participate in public conversations to help support and remind community members about these expectations, and help to model more ideal interactions. They will also work behind the scenes to give feedback and guidance to users on how their messaging is being received by different groups. 

 Repeated, outright refusal to adopt these community norms, especially in the more formal channels including but not limited to those outlined above, may escalate into unacceptable territory and consequences will be enforced. 
 

>  However, members who are open to the feedback but struggle to adopt it as their own style of communication due will not be removed unless their actions cross over into those on the "unacceptable behaviour" list. This is necessary to remain inclusive to members of our community who have difficulty reading social cues due to ability, language barriers, cultural differences, differences in lived experiences or just the ambiguity of internet communication. 

Reviewer: The `due` in this threw me off, should this instead read as:

```suggestion
 However, members who are open to the feedback but struggle to adopt it as their own style of communication will not be removed unless their actions cross over into those on the "unacceptable behaviour" list. This is necessary to remain inclusive to members of our community who have difficulty reading social cues due to ability, language barriers, cultural differences, differences in lived experiences or just the ambiguity of internet communication. 
```

?

## 4\. Unacceptable Behaviour

The following behaviors are considered harassment and are unacceptable anywhere within our community, and will result in a moderator reaching out as outlined in the consequences section. This is not an exhaustive list of unacceptable behaviour. This extends to any events held by the community. 

* Using your position of authority (e.g. leadership role, or social status, or moderator position) to minimize, shut down or otherwise discourage other members of the community (e.g. more junior or minority members) from advocating for what they need to thrive. 
* Violence, threats of harm or violent language directed against another person.
* Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
* Posting or displaying sexually explicit or violent material.
* Posting or threatening to post other people's personally identifying information ("doxing").
* Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.

> * Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances (e.g DMing a member of the community with the intent to persue a relationship).

Reviewer: Typo:

```suggestion
* Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcomed sexual advances (e.g DMing a member of the community with the intent to pursue a relationship).
```

* Deliberate intimidation, stalking or following (online or in person).
* Sustained disruption of community events, including talks and presentations.

> * Inappropriate physical contact. You should have someone's consent before touching them.

Reviewer: Would suggest stronger wording:

```suggestion
* Inappropriate physical contact. You must have someone's consent before touching them.
```

Author: thank you!

* Inappropriate photography or recording.
* Advocating for, or encouraging, any of the above behavior.

Unacceptable behavior can also include various kinds of "bad faith participation", including such examples as:

> * Starting conversations with the express purpose of having an argument or debating a topic; This is a forum for casual professional networking, this is not IRC or Reddit. Please keep debates confined to the #debate channel.

Reviewer: Maybe include link to the #debate channel:

```suggestion
* Starting conversations with the express purpose of having an argument or debating a topic; This is a forum for casual professional networking, this is not IRC or Reddit. Please keep debates confined to the [#debate channel](https://app.slack.com/client/T18HH1BSP/CB3V1CTPH).
```
